src/__tests__/pages/api/auth/signup.test.ts

```diff
@@ -1,32 +1,21 @@
 /**
  * @jest-environment node
  */
-import * as dotenv from 'dotenv';
-dotenv.config();
-
 import { PrismaClient } from '@prisma/client';
+import { NextApiRequest, NextApiResponse } from 'next';
 import signUpHandler from '../../../../pages/api/auth/signup';
 import {
   mockUserData,
   mockManyUserData
 } from '../../../../__mocks__/pages/api/auth';
 import type { UserDataProp } from '../../../../__mocks__/pages/api/auth';
-import { NextApiRequest, NextApiResponse } from 'next';
 
 const prisma = new PrismaClient();
 
-beforeAll(async () => {
-  await prisma.$connect();
```

Comment:
We don't need the `$connect`/`$disconnect` anymore?

Reply:
Since we have `--runInBand` and we reset the database after every run, we can get rid of it (I'm not sure about this lol, I removed but it still works)